chains/chronicleyellowstone/metadata.yaml

```
@@ -0,0 +1,29 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://yellowstone-explorer.litprotocol.com/api
+    family: etherscan
+    name: Yellowstone Explorer
+    url: https://yellowstone-explorer.litprotocol.com
+blocks:
+  confirmations: 1
+  estimateBlockTime: 1
+  reorgPeriod: 0
+chainId: 175188
+deployer:
+  name: Abacus Works
+  url: https://www.hyperlane.xyz
+displayName: Chronicle Yellowstone
+domainId: 175188
+gasCurrencyCoinGeckoId: ???
+index:
+  from: ??? # Index of Mailbox deployment, nitro specific
```

just grabbed the latest block number for now, will update when we do our deploy
```suggestion
  from: 2339059
```